api.ts

```diff
@@ -135,9 +135,14 @@ export interface SourceFileConfiguration {
     readonly intelliSenseMode: "msvc-x64" | "gcc-x64" | "clang-x64";
     
     /**
-     * The C or C++ standard to emulate.
+     * The C standard to emulate.
      */
-    readonly standard: "c89" | "c99" | "c11" | "c++98" | "c++03" | "c++11" | "c++14" | "c++17";

```

Comment:
A source file cannot be treated as both C and C++. That is why the field is named the way it is.